scorecard/tests/indicators/resources.py

```diff
@@ -21,28 +27,6 @@ class Meta:
         import_id_fields = ['geo_code']
 
 
-class IncexpFactsV1Resource(resources.ModelResource):
```

Comment:
did you move this out of tests because you thought initially you're able to use import-export?

Reply:
@jbothma same answer as the previous one.